benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```
@@ -168,6 +213,17 @@
     "PageFieldDisplayOrder": 3,
     "PageID": "1.0"
   },
+  {
+    "LiveFrom": "06/10/2022",
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "reserveTo",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
```

This has been changed to complex for https://tools.hmcts.net/jira/browse/SSCS-10822
```suggestion
    "DisplayContext": "COMPLEX",
```

---

Can wait for 10822